src/Win32/InternalsVisibleTo.cs

```diff
@@ -2,6 +2,10 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Limit P/Invoke to assemblies located in the System32 folder
```

Comment:
I see now why you wanted to rename this file. The floor is open.

Reply:
Per our conversation, file is again named AssemblyInfo.cs. I checked docs\AssessingInternals.md, and that file already says AssemblyInfo.cs